dotnetv3/SNS/GetTopicAttributesExample/GetTopicAttributesExample/GetTopicAttributes.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+namespace GetTopicAttributesExample
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon.SimpleNotificationService;
+    using Amazon.SimpleNotificationService.Model;
+
+    // snippet-start:[SNS.dotnetv3.GetTopicAttributesExample]
+
+    /// <summary>
+    /// This example shows how to retrieve the attributes of an Amazon Simple
+    /// Notification Service (Amazon SNS) topic. The example was writing using
+    /// the AWS SDK for .NET 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class GetTopicAttributes
+    {
+        public static async Task Main()
+        {
+            string topicArn = "arn:aws:sns:us-west-2:000000000000:ExampleSNSTopic";
+            IAmazonSimpleNotificationService client = new AmazonSimpleNotificationServiceClient();
+
+            var attributes = await GetTopicAttributesAsync(client, topicArn);
+            DisplayTopicAttributes(attributes);
+        }
+
+        /// <summary>
+        /// Given the ARN of the SNS topic, this method retrieves the topic
+        /// attributes.
+        /// </summary>
+        /// <param name="client">The initialized SNS client object used to
+        /// retrieve the attributes for SNS topic.</param>
```

Comment:
for **the** SNS topic

Reply:
fixed - also change SNS topic to Amazon SNS topic wherever it occurs.